ports/geckolib/glue.rs

```diff
@@ -535,12 +545,12 @@ pub extern "C" fn Servo_AnimationCompose(raw_value_map: RawServoAnimationValueMa
         return;
     }
 
-    let position = unsafe {
+    let progress = unsafe {
```

Comment:
Why? It makes the code more verbose, and this quantity is called progress on Servo's side.